build.gradle

```diff
@@ -17,7 +17,7 @@ repositories {
 sourceCompatibility = '1.8'
 targetCompatibility = '1.8'
 
-def versionsMap = ['IU-2019.2':'192.5728.98','IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152']
+def versionsMap = ['IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152', 'IU-2021.1':'211.6693.108']
```

Comment:
Out of curiosity, what exactly is the reasoning for you to pick the IJ version to use? I can see in the website the newest one is 211.6693.111, why do you use 211.6693.108?

Reply:
I'm using this as the source: https://plugins.jetbrains.com/plugin/10485-kubernetes/versions